pyscript.core/esm/worker/_template.js

```diff
@@ -23,15 +23,15 @@ try {
     );
 }
 
-let engine, run, interpreterEvent;
+let interpreter, run, interpreterEvent;
 const add = (type, fn) => {
     addEventListener(
         type,
         fn ||
             (async (event) => {
-                const interpreter = await engine;
+                await interpreter;
                 interpreterEvent = event;
-                run(interpreter, `xworker.on${type}(xworker.event);`, xworker);
+                run(`xworker.on${type}(xworker.event);`, xworker);
```

Comment:
we don't need the interpreter here?

Reply:
it's bound once and "forever" in `run = details[name].bind(details, interpreter);`